Source/Extensions/Blazorise.DataGrid/_DataGridCellDatePicker.razor.cs

```diff
@@ -13,18 +14,34 @@ namespace Blazorise.DataGrid;
 /// <typeparam name="TItem"></typeparam>
 public partial class _DataGridCellDatePicker<TItem> : ComponentBase
 {
+    #region Members
+
     protected string elementId;
 
     /// <summary>
     /// Value data type.
     /// </summary>
     private Type valueType;
 
+    private string dateDisplayFormat;
+
+    #endregion
+
+    #region Methods
+
     /// <inheritdoc/>
     protected override void OnInitialized()
     {
         valueType = Column.GetValueType( default );
         elementId = IdGenerator.Generate;
+
+        // this is a woraround for https://github.com/Megabit/Blazorise/issues/5837
+        // in 2.0 we need to remove formating as {0:dd.MM.yyyy}, and on support dd.MM.yyyy
+        if ( Column.DisplayFormat is not null && Column.DisplayFormat.StartsWith( "{0:" ) && Column.DisplayFormat.EndsWith( "}" ) )
+        {
+            dateDisplayFormat = Column.DisplayFormat.Substring( 3, Column.DisplayFormat.Length - 4 );
```

Comment:
`dateDisplayFormat = Column.DisplayFormat.Substring( 3, Column.DisplayFormat.Length - 4 );`

Shouldn't it be **Column.DisplayFormat.Length - 1**? Or Am I missing something?

Reply:
No, Substring starts from index, and then takes the length.

Reply:
Yeah range works like that. I always forget about the range :)